Sources/SwifterSwift/UIKit/UIImageExtensions.swift

```diff
@@ -201,6 +201,28 @@ public extension UIImage {
         return UIGraphicsGetImageFromCurrentImageContext()!
     }
 
+    /// SwifterSwift: UImage with background color
+    ///
+    /// - Parameters:
+    ///   - backgroundColor: Color to use as background color
+    /// - Returns: UIImage with a background color that is visible where alpha < 1
+    func withBackgroundColor(_ backgroundColor: UIColor) -> UIImage? {
+        let rect = CGRect(origin: .zero, size: size)
+        UIGraphicsBeginImageContextWithOptions(rect.size, false, scale)
```

Comment:
This is an old API, and although is not deprecated I think is not recommended to use it anymore. I know all the other extensions use that (we maybe change those in the future) is preferable for new extensions to use `UIGraphicsImageRenderer` API instead because unlike `UIGraphicsBeginImageContextWithOptions`, `UIGraphicsImageRenderer` is capable of rendering wide-color content correctly.

Reply:
I agree. However, UIGraphicsImageRenderer is only available in iOS 10 and later. I could provide a fallback, but the result would still be `UIImage?` instead of `UIImage` which would be possible with UIGraphicsImageRenderer.
What are your thoughts about this?

Reply:
I'm fine with providing this API only for iOS 10+, but feel free to provide a fallback if you think it adds value. I personally think that fewer people are supporting less than iOS 10 these days so it wouldn't affect too much :))

Reply:
I'm all for using `UIGraphicsImageRenderer` with a backwards-compatible fallback, I think it would be the right way to go about this.